xml/tuning_tuned.xml

```diff
@@ -0,0 +1,2501 @@
+    </para>
+
+    <variablelist>
+      <varlistentry>
+        <term>&tuned; daemon</term>
+        <listitem>
+          <para>
+            The core daemon process that runs in the background, monitoring system usage and
+            applying tuning settings. It handles profile switching and coordination of the other
+            components. The following items are associated with the daemon:
+          </para>
+          <itemizedlist>
+            <listitem>
+              <para>
+                The <command>tuned</command> command that you can use after enabling &tunedapp;
+                using <command>systemctl</command>. When invoked from the terminal manually, by
+                default <command>tuned</command> runs as a normal process. However, you can use the
+                <option>--daemon</option> option to run it as a background process.
+              </para>
+            </listitem>
+            <listitem>
+              <para>
+                A <emphasis>tuned</emphasis> systemd service that manages the daemon process, such
+                as starting the daemon at boot time, stopping or restarting it, and ensuring it
+                runs with the necessary permissions and environment.
+              </para>
+            </listitem>
+          </itemizedlist>
+          <para>
+            For more information, read the man page for &tuned;.
+          </para>
+<screen>&prompt.user;<command>man tuned</command></screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>tuned-adm</term>
+        <listitem>
+          <para>
+            A command-line utility to manage or administer the &tuned; daemon. To understand the
+            basics of managing the &tuned; daemon using <command>tuned-adm</command>, see the
+            section <xref linkend="sec-tuning-use-tuned-adm"></xref>.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Profiles</term>
+        <listitem>
+          <para>
+            A &tunedapp; <emphasis>profile</emphasis> is a collection of settings for system
+            components that you can apply to tune the system. System administrators can either use
+            the supported profiles that are installed at <filename>/usr/lib/tuned/</filename> as
+            part of the <package>tuned</package> package, or apply customized profiles by creating
+            them at <filename>/etc/tuned/</filename>. If the filenames for a custom and a supported
+            profile matches, the custom profile takes precedence when applied. For more
+            information, see the section <xref linkend="sec-tuning-tuned-profiles"></xref>.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Plug-ins</term>
+        <listitem>
+          <para>
+            A &tunedapp; <emphasis>plug-in</emphasis> is an implementation of tuning logic for
```

Comment:
Maybe consider to use `emphasis` around the `tunedapp` as well? I think it's not a general plug-in, but a tunedapp plug-in, isn't it?

```suggestion
            A <emphasis>&tunedapp; plug-in</emphasis> is an implementation of tuning logic for
```